plugin/providers/phpipam/custom_field_structure.go

```diff
@@ -114,8 +116,13 @@ func updateCustomFields(d *schema.ResourceData, client interface{}) error {
 		panic(fmt.Errorf("Invalid client type passed %#v - this is a bug", client))
 	}
 	if err != nil {
-		return fmt.Errorf("Error getting custom fields for updating: %s", err)
+		if strings.Contains(err.Error(), "404") && len(customFields) == 0 {
```

Comment:
I propose to change this condition to next to handle error from old phpipam versions:
```
if (strings.Contains(err.Error(), "404") || (strings.Contains(err.Error(), "200") && strings.Contains(err.Error(), "No custom fields defined")) && len(customFields) == 0 {
```

Reply:
Fine by me. I won't have the time to test it properly in the foreseeable future though.